openreview/venue/webfield/publicationChairWebfield.js

```diff
@@ -0,0 +1,30 @@
+// Webfield component
```

Comment:
@haroldrubio Here I am adding the webfield, it only has an 'Accepted Submissions' tab, so if publication chairs are readers of accepted submissions they will see them here.